crates/ef-testing/tests/tests.rs

```diff
@@ -127,6 +127,9 @@ mod blockchain_tests {
     // blockchain_tests!(st_transition_test, stTransitionTest); // 3 failing tests
     blockchain_tests!(st_wallet_test, stWalletTest);
     blockchain_tests!(st_zero_calls_revert, stZeroCallsRevert);
+    // blockchain_tests!(st_zero_calls_test, stZeroCallsTest); // 1 failing test
+    blockchain_tests!(st_zero_knowledge, stZeroKnowledge);
+    blockchain_tests!(st_zero_knowledge2, stZeroKnowledge2);
     blockchain_tests!(st_zero_calls_test, stZeroCallsTest);
     // blockchain_tests!(st_zero_knowledge, stZeroKnowledge); // 60 failing
```

Comment:
These tests have been duplicated

Reply:
stZeroKnowledge and stZeroKnowledge2 are 2 different tests, what do you mean?

Reply:
Here you have:

    blockchain_tests!(st_zero_knowledge, stZeroKnowledge);

and 

    // blockchain_tests!(st_zero_knowledge, stZeroKnowledge); // 60 failing

If we implement a test we should just remove the comment, not add another line